packages/create-app/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Can you use comments in JSON? (Github seems to be annoyed anyway)

Reply:
this is for vscode